libs/experimental/tests/integration_tests/llms/test_ollama_functions.py

```diff
@@ -22,7 +24,7 @@ class TestOllamaFunctions(unittest.TestCase):
     """
 
     def test_default_ollama_functions(self) -> None:
-        base_model = OllamaFunctions(model="llama3", format="json")
+        base_model = OllamaFunctions(model="phi3", format="json")
```

Comment:
why make this change?

Reply:
phi3 generations for tool calling seem to be more stable than llama3 which on some occasions fail to produce valid json or produce incorrect json.